bot/exts/evergreen/githubinfo.py

```diff
@@ -0,0 +1,88 @@
+import logging
+from datetime import datetime
+
+import aiohttp
+import discord
+from discord.ext import commands
+from discord.ext.commands.cooldowns import BucketType
+
+log = logging.getLogger(__name__)
+
+
+class GithubInfo(commands.Cog):
+    """Fetches GitHub."""
+
+    def __init__(self, bot: commands.Bot):
+        self.bot = bot
+
+    async def fetch_data(self, url: str) -> dict:
```

Comment:
```suggestion

    @staticmethod
    async def fetch_data(url: str) -> dict:
```
The method isn't using the `self` attribute and can therefore be a static method.